HACKER/HACKER/Sources/ViewControllers/ShoppingRankingViewController/Cells/ShoppingCollectionViewCell.swift

```diff
@@ -0,0 +1,108 @@
+//
+//  ShoppingCollectionViewCell.swift
+//  HACKER
+//
+//  Created by 김지수 on 2022/03/26.
+//
+
+import UIKit
+
+import SnapKit
+import Then
+
+// MARK: - ShoppingCollectionViewCell
+class ShoppingCollectionViewCell: UICollectionViewCell {
+  
+  // MARK: - Components
+  let myFriendsCollectionView: UICollectionView = {
+    let layout = UICollectionViewFlowLayout()
+    layout.scrollDirection = .vertical
+    layout.minimumLineSpacing = 0
+    layout.minimumInteritemSpacing = 0
+    let collectionView = UICollectionView(frame: .zero, collectionViewLayout: layout)
+    collectionView.isScrollEnabled = true
+    collectionView.translatesAutoresizingMaskIntoConstraints = false
+    return collectionView
+  }()
+  var closure: (() -> Void)?
+  
+  // MARK: - LifeCycle
+  override func awakeFromNib() {
+    super.awakeFromNib()
+    register()
+    attribute()
+    layout()
+  }
+}
+// MARK: - Extensions
+extension ShoppingCollectionViewCell {
+  func register() {
+    self.myFriendsCollectionView.register(AddFriendCollectionViewCell.self, forCellWithReuseIdentifier: AddFriendCollectionViewCell.identifier)
+    self.myFriendsCollectionView.register(MyFriendsCollectionViewCell.self, forCellWithReuseIdentifier: MyFriendsCollectionViewCell.identifier)
+  }
+  func attribute() {
+    self.myFriendsCollectionView.delegate = self
+    self.myFriendsCollectionView.dataSource = self
+  }
+  func layout() {
+    layoutMyFriendsCollectionView()
+  }
+  func layoutMyFriendsCollectionView() {
+    self.contentView.add(myFriendsCollectionView) {
+      $0.backgroundColor = .hackerWhite
+      $0.isUserInteractionEnabled = true
+      $0.showsVerticalScrollIndicator = false
+      $0.snp.makeConstraints { make in
+        make.top.equalToSuperview().offset(10)
```

Comment:
```suggestion
        make.top.equalToSuperview()
```

여기 offset을 10 대신 0으로 설정하고, 아래에서 collectionview 자체에 inset으로 top 10을 주면
컬렉션뷰 스크롤 영역이 늘어나서 스크롤 했을 때 세그(구경하기, 랭킹) 아래 구분선이랑 스크롤 영역 사이의 공간을 줄일 수 있어
그럼 좀 더 자연스러울 거 같애!!

|Before|After|
|-|-|
|<img width="100%" alt="image" src="https://user-images.githubusercontent.com/71601985/160801906-7355ac11-af17-4a23-b29f-c770e068d33c.png">|<img width="100%" alt="image" src="https://user-images.githubusercontent.com/71601985/160801568-5cb17250-1aec-4472-9ed8-588d71f3d96b.png">|

<br/>

collectionView inset은 아래처럼 주면 되는데 코드 있는 곳 가서 코드리뷰 suggestion code로 다시 써줄게!
```swift
extension ShoppingCollectionViewCell: UICollectionViewDelegateFlowLayout {
  func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, insetForSectionAt section: Int) -> UIEdgeInsets {
    return UIEdgeInsets(top: 10, left: 0, bottom: 10, right: 0)
  }
}
```